swap_meet/clothing.py

```diff
@@ -0,0 +1,13 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "", condition = 0):
+        # super().__init__(category = "Clothing")
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
Another way to write this is to use super to take advantage of the code in Item's constructor (also, since Clothing always has the category "Clothing", the parameter `category` in the constructor can be eliminated):
```
def __init__(self, condition = 0):
    super().__init__("Clothing", condition)
```